reflekt-plugin/src/main/kotlin/io/reflekt/plugin/generation/bytecode/ReflektGeneratorExtension.kt

```diff
@@ -0,0 +1,92 @@
+package io.reflekt.plugin.generation.bytecode
```

Comment:
I think we should add some comments to this file, especially in `ReflektInvokeNames` class.